src/ipc/commands.rs

```diff
@@ -37,4 +37,17 @@ pub enum Command {
         /// The path to the sheet.
         path: PathBuf,
     },
+
+    /// Set the visibility of the specified window with the given name.
+    SetVisible {
+        ///Bar name to target.
+        bar_name: String,
+        /// The visibility status.
+        visible: bool,
+    },
+    /// Get the visibility of the specified window with the given name.
```

Comment:
Please add whitespace between lines:

```suggestion

    /// Get the visibility of the specified window with the given name.
```